serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -65,8 +70,10 @@ include::modules/kn-trigger-describe.adoc[leveloffset=+1]
 .Procedure
 
 To delete a trigger, enter the following command:
+
+[source,terminal]
```

Comment:
I see you adding [source,terminal] a few places, but in looking through the entire serverless book, there are a ton that don't have this yet. I might suggest to not worry about that in this PR, and then open a PR after this is merged to add those in.

Reply:
Cool, I'll leave the ones I've added in this PR as are though if that's OK? :slightly_smiling_face: Or do I need to revert them?